README.md

```diff
@@ -1,117 +1,65 @@
-# Jenkins Account Management/Sign-up App
+# Jenkins Account Management
 
 ## Testing locally
 
-First, set up a tunnel to Jenkins LDAP server. Run the following command and
-keep the terminal open:
+Run `docker compose up -d ldap`
 
 ```shell
-ssh -4 -L 9389:localhost:389 ldap.jenkins.io
+./gradlew appRun
 ```
 
-Create `config.properties` in the same directory as `pom.xml`. See the
-`Parameters` class for the details, but it should look something like the
-following:
-
-```init
-server=ldap://localhost:9389/
-managerDN=cn=admin,dc=jenkins-ci,dc=org
-newUserBaseDN=ou=people,dc=jenkins-ci,dc=org
-smtpServer=localhost
-managerPassword=*****
-circuitBreakerFile=./circuitBreaker.txt
-url=http://localhost:8080/account/
-```
+This will get you a development server running at `http://localhost:8080`
+
+## Testing locally against production
 
-Finally, run the application with Jetty, then access `http://localhost:8080/`:
+If you want to use real data you will need to set a few environment variables:
 
 ```shell
-./gradlew -Djira.url=https://issues.jenkins.io/ -Djira.username=kohsuke -Djira.password=... -Durl=ldap://localhost:9389 -Dpassword=... jettyRun
+JIRA_USERNAME=<insert your jira username>
+JIRA_PASSWORD=<insert your jira password>
+JIRA_URL=https://issues.jenkins.io
+LDAP_URL=ldap://localhost:1389/ # needs to be port forwarded see below
+LDAP_PASSWORD=<insert the ldap admin password>
+LDAP_MANAGER_DN=cn=admin,dc=jenkins-ci,dc=org
+SMTP_SERVER=smtp.sendgrid.net
+SMTP_AUTH=true
+SMTP_USER=apikey
+SMTP_PASSWORD=<insert sendgrid api key>
 ```
 
-(As you can see above, this connects your test instance to the actual LDAP
-server, so the data you'll be seeing is real.
-
-The command line system properties are for JIRA LDAP sync tool. JIRA user account you are providing has to have the system admin access to JIRA.
-TODO: feed this data from config.properties
-
-### Docker Compose
-
-A docker compose file can be used for testing purpose.
+and port forward to the production ldap instance:
```

Comment:
This is a recipe for disaster 😱 

Is there a way to use the docker-ldap image instead to provide a working environment?

Reply:
This is what the README previously documented just with the old setup.

I only maintained the docs for the new setup, I’m happy to remove it not attached to it at all

Reply:
the docs I added also reference using mock-ldap which is a working ldap container and the expected default dev environment